src/Components/test/testassets/BasicTestApp/Index.razor

```diff
@@ -27,7 +27,7 @@
         <option value="BasicTestApp.ErrorComponent">Error throwing</option>
         <option value="BasicTestApp.EventBubblingComponent">Event bubbling</option>
         <option value="BasicTestApp.EventCallbackTest.EventCallbackCases">EventCallback</option>
-        <option value="BasicTestApp.EventCasesComponent">Event cases</option>
+        <option value="BasicTestApp.EventCustomArgsComponent">Event custom arguments</option>
```

Comment:
I'm not renaming or replacing here, even though it looks like it in the diff.

I'm removing `EventCasesComponent` because it doesn't exist and hasn't done for a long time - this is an orphaned link. And separately, I'm adding `EventCustomArgsComponent`.